sklearn/tests/test_pipeline.py

```diff
@@ -1072,3 +1072,9 @@ def test_make_pipeline_memory():
     assert len(pipeline) == 2
 
     shutil.rmtree(cachedir)
+
+
+def test_pipeline_param_error():
+    clf = make_pipeline(LogisticRegression())
+    with pytest.raises(TypeError, match='sample_weight'):
```

Comment:
`match='Pipeline.fit does not accept the sample_weight parameter'`

would be enough